src/app/utils/field-validation/validators/dateValidator.ts

```diff
@@ -191,7 +191,9 @@ const pastOnlyValidatorV3: ResponseValidator<DateResponseV3> = (response) => {
   const answerDate = createMomentFromDateStringV3(answer)
 
   return answerDate.isAfter(todayMax)
-    ? left(`DateValidatorV3:\t answer does not pass date logic validation`)
+    ? left(
```

Comment:
The changes in this file are added to make the logs more specific - not related to the skipping of validation